BlueprintToCpp/Program.cs

```diff
@@ -87,16 +87,25 @@ public static async Task Main(string[] args)
                 return;
             }
 
+            string zlibPath = config.ZlibPath;
+            if (string.IsNullOrEmpty(zlibPath) || zlibPath.Length < 1)
+            {
+                Console.WriteLine("Missing \"ZlibPath\" in the config.json file.");
+            }
```

Comment:
This part could be removed if you don't want the program to warn that ZlibPath is missing. The program will still run if ZlibPath is missing. This just warns the user if it's missing.
```suggestion
```